Source/WebGPU/WGSL/ConstantValue.h

```diff
@@ -36,22 +36,38 @@ namespace WGSL {
 #if HAVE(FP16_HALF_SUPPORT)
 using half = __fp16;
 #else
+// Wrap _Float16, which is the 16bit float type in C++23
```

Comment:
I wonder if you could just use `<wtf/Float16.h>` instead, but for some reason it looks like `HAVE(FLOAT16)` is only 1 on `arm64`?

Reply:
Right. HAVE_FP16_HALF_SUPPORT is defined if and only if CPU is `arm64`.

I didn't want to overly complicate the patch by doing a more granular check for the specific feature.